operators/pkg/instctrl/containers.go

```diff
@@ -89,6 +139,22 @@ func (r *InstanceReconciler) enforceContainer(ctx context.Context) error {
 		// either rejected or cause the restart of the Pod, with consequent possible data loss.
 		if depl.CreationTimestamp.IsZero() {
 			depl.Spec = forge.DeploymentSpec(instance, environment, &r.ContainerEnvOpts)
+
+			if environment.Mode == v1alpha2.ModeStandard && environment.MountPathEnabled && instance.Spec.Tenant.Name != "" {
```

Comment:
I'm still not too sure this is the best approach. I'd directly try retrieval of the secret in the current instance NS, regardless of the presence of the tenant name. Also, I'd set as the only check condition, the presence of `environment.MountPathEnabled`. Special Modes could still be able to mount the shared storage...

Reply:
Ok, removed checks for mode and tenant name